.cirrus.yaml

```
@@ -0,0 +1,10 @@
+test_task:
+  name: "Test"
+  persistent_worker:
+    labels:
+      os: darwin
+      arch: arm64
+      purpose: stage
+    test_script: |
```

```suggestion
  test_script: |
```

Need to be intended on the task level, i.e. same level as persistent_worker